diff --git a/node_modules/shp-write/src/zip.js b/node_modules/shp-write/src/zip.js
index b8e89d8..e572c7b 100644
--- a/node_modules/shp-write/src/zip.js
+++ b/node_modules/shp-write/src/zip.js
@@ -30,9 +30,6 @@ module.exports = function(gj, options) {
 
     var generateOptions = { compression:'STORE' };
 
-    if (!process.browser) {
-      generateOptions.type = 'nodebuffer';
-    }
 
     return zip.generate(generateOptions);
 };
